src/common/components/help/help-install-snap.js

```diff
@@ -25,15 +25,15 @@ export default class HelpInstallSnap extends Component {
           </p>
         }
         <p className={ styles.p }>
-          Don’t have snapd installed?  {' '}
+          (
           <a
-            className={ styles.external }
```

Comment:
Spec is not a visual design, so I think we should keep external link icon - this is a link outside of BSI (and opens in new tab), to it makes sense to keep the icon here.